client/src/components/NavLink/NavLink.css

```diff
@@ -0,0 +1,34 @@
+.NavLink-Link {
```

Comment:
Can you update your classnames to use the BEM naming convention? There's documentation in the readme for it now